test/fail_compilation/diag13215.d

```diff
@@ -0,0 +1,12 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/diag13215.d(11): Error: cannot implicitly convert expression `[1, 2, 3]` of type `int[]` to `immutable(uint[2])[]`
+---
```

Comment:
Although the bugzilla issue number is implicit in the filename, it is nice to include a clickable URL to it as well in the comment.